modules/tinymce/src/core/main/ts/init/InitIframe.ts

```diff
@@ -83,11 +83,10 @@ const getIframeHtml = (editor: Editor) => {
 };
 
 const createIframe = (editor: Editor, o) => {
-  const title = editor.editorManager.translate(
-    'Rich Text Area. Press ALT-0 for help.'
-  );
+  const iframeTitle = editor.getParam('content_aria_label', 'Rich Text Area. Press ALT-0 for help.');
+  const iframeTranslatedTitle = editor.editorManager.translate(iframeTitle);
```

Comment:
```suggestion
  const iframeTranslatedTitle = editor.translate(iframeTitle);
```
I know this was like this previously, but it can be simplified now :shrug: